molecule/oauth-rbac-plain-provided-debian12/verify.yml

```diff
@@ -135,9 +135,38 @@
         validate_certs: false
         url_username: "kafka_connect"
         url_password: "password"
+      register: connectors1
+
+    - debug:
+        msg: "{{connectors1.json}}"
+
+    - import_role:
+        name: confluent.platform.common
+        tasks_from: get_authorization_token.yml
+      vars:
+        oauth: "{{ kafka_connect_oauth_enabled }}"
+        oauth_user: "{{ kafka_connect_oauth_user }}"
+        oauth_password: "{{ kafka_connect_oauth_password }}"
+        ldap_user: "{{ mds_super_user }}"
```

Comment:
shouldn't this be kafka_connect_ldap_user?

Reply:
Yes will update it